.github/workflows/tests.yml

```diff
@@ -8,7 +8,11 @@ jobs:
   test-bot:
     strategy:
       matrix:
-        os: [ubuntu-22.04, macos-12]
+        os:
+          - ubuntu-22.04
+          - macos-12
```

Comment:
Did you explicitly not choose ubuntu/macos-latest here? I wouldn't want to constantly bump this when we just want to use the latest anyway?

Reply:
This file was originally created by `brew tap-new` and used explicit versions. But I agree that latest makes more sense.